.github/workflows/admin.yml

```diff
@@ -1,11 +1,14 @@
 name: Repo Admin
 on:
-  pull_request_target:
-    types:
-      - opened
-      - edited
-      - synchronize
-      - reopened
+  # TODO(!!!): Remove before merge
```

Comment:
Specifically flagging this line (and the ones below). See this section in the `actions/labeler` README: https://github.com/actions/labeler?tab=readme-ov-file#updating-major-version-of-the-labeler